Modelica/Blocks/Logical.mo

```diff
@@ -973,6 +973,41 @@ parameter \"terminationText\".
 
 </html>"));
   end TerminateSimulation;
+
+  block AssertCondition "Trigger error (or warning) and print message if assertion condition is not fulfilled"
+
+    Modelica.Blocks.Interfaces.BooleanOutput condition=true
+      "Trigger error (or warning) and print message when condition becomes false"
+      annotation (Dialog, Placement(transformation(extent={{200,-10},{220,10}})));
+    parameter String message="... assertion condition no longer true"
+      "Text that will be displayed when assertion condition is false";
+    parameter AssertionLevel assertionLevel=AssertionLevel.error
+      "Select warning (1) or error (2)";
```

Comment:
What are the numbers in parentheses supposed to mean? Wouldn't this be more to the point?
```suggestion
      "Assertion level";
```

Reply:
> What are the numbers in parentheses supposed to mean? Wouldn't this be more to the point?

Assertion level is an enumeration, so after compilation in an FMU for example, the integer 1 corresponds to warning and 2 corresponds to error. I could go either way, but in some of my personal usage of enumeration parameters, I found this useful. Open to suggestions.

Reply:
But FMI 1 for ModelExchange and FMI 2 can describe assertion-types for an FMU, so even in that case it shouldn't be needed.